src/lxc/macro.h

```diff
@@ -27,6 +27,7 @@
 #include <linux/loop.h>
 #include <linux/netlink.h>
 #include <linux/rtnetlink.h>
+#include <stdbool.h>
```

Comment:
Is this a leftover? Not required `_exit()` at least, maybe also as part of the `macro...` commit?